nf_core/pipeline-template/nextflow.config

```diff
@@ -77,11 +77,6 @@ params {
 includeConfig 'conf/base.config'
 {%- else %}
 
-{% if modules -%}
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
```

Comment:
Are we now not loading modules 
.config anywhere anymore?

Reply:
it is at the end https://github.com/nf-core/tools/pull/3369/files#diff-957ebc7549e67b6aa1df19ae402551fee3e2e06afbc7c563424a4515b1434627R337